api/management/commands/script.py

```diff
@@ -0,0 +1,27 @@
+from django.core.management.base import BaseCommand
+from api.models import ScheduledActivity
+from api.external_services.geocode_service import GeocodeService
+from api.external_services.darksky_service import DarkskyService
+from api.helpers import *
+from datetime import datetime
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        current_date = datetime.now().strftime('%Y-%m-%d')
+        scheduled_activities = ScheduledActivity.objects.filter(date__gte=current_date)
```

Comment:
Is python able to evaluate if a string date "YYYY-MM-DD" is greater than or equal to a `current_date` object?

Reply:
Yes I was surprised too but tested different cases and it works